tests/test_my_project.py

```diff
@@ -27,3 +33,115 @@ def test_python_dot_org(driver: WebDriver):
 
     # Assert that the first search result is some text...
     assert first_result.text == 'PEP 394 -- The "python" Command on Unix-Like Systems'
+
```

Comment:
I would suggest creating a new file and naming menu bar tests. Its best to keep the demo test by itself to not confuse what the file tests are about

Reply:
since we are can write tests with Selenium, there is no need for the demo anymore so I will remove it demo test and also will change the file name

Reply:
The demo is just there to show a basic test, it can be removed when you write your own tests (those tests can be the new "demo" to show how to write tests)